code/modules/clothing/gloves/color.dm

```diff
@@ -10,8 +10,8 @@
 	permeability_coefficient = 0.05
 	resistance_flags = NONE
 	var/can_be_cut = 1
-	custom_price = 1200
-	custom_premium_price = 1200
+	custom_price = 400 //powergaming isn't cheap
+	custom_premium_price = 400
```

Comment:
I'd say `PAYCHECK_MEDIUM * STARTING_PAYCHECKS + 25` (900) is a better price, so the average non-greyshirt will always be a few credits shy from buying those gloves roundstart. :smiling_imp: